NO_RODE_001 (#361)
NINAnor/ecRxiv: R/NO_RODE_001.qmd

  dplyr::filter(Variable %in% c(
    "Indicator ID",
    "Indicator Name",
    "Continent",
    "Country",
    "Ecosystem Condition Typology Class", 
    "Realm",
    "Biome",
    "Ecosystem",
    "Year added", 
    "Last update",
    "Version",
    "Version comment",
    "Normalised",
    "Spatial aggregation pathway")) |>   
  kable()

```

:::

::: {.callout-tip collapse="true"}

## Log

<!--# Update this logg with short messages for each update -->
- May 28, 2026 - Started work
:::

<hr />

<!--# Document you work below.  -->

```{r fig-leming}
#| fig-cap: 'Lemming. Photo: John Linnell.'
knitr::include_graphics(here::here("img/lemen_linnell_cropped.jpg"))
```
## 1. Summary

<!--# 
With a maximum of 300 words, describe the indicator in general terms as you would to a non-expert. Think of this as a kind of common language summary. Follow the four numbers sub-headers below. under point 2, it is possible under  include a bullet point list of the specific steps in the workflow. 
 -->

1. **Background and rationale**:
<!-- What does the metric represent? Why is this relevant for describing ecosystem condition in this ecosystem?
What are the main anthropogenic impact factors? -->

> Small rodents, like lemming and mice, are keystone species in aline ecosystems. They function as a trophic link between the primary producers and top predators. When population are high, they impose a large disturbance on the vegetation, but at the same time they fuel the higher troficl levels and ensure survival and breeding success for species like raptors and foxes. Rodent cycles have a frequency of about 3-4 years, and it is the amplitude (size or height) of these peaks that make up this indicator. Rodent cycles, both their regularity and their amplidude, are negatively affected by many known and unknown drivers. The most important antropogenic driver is climate change, with warmer and wetter winters causing rodent declines due to the effect that this has on the snow cover.  

Reviewer: ```suggestion
Small rodents, such as lemmings and mice, are keystone species in alpine ecosystems. They function as a trophic link between the primary producers and top predators. When population are high, they impose a large disturbance on the vegetation, but at the same time they fuel the higher trophic levels and ensure survival and breeding success for species like raptors and foxes. Rodent cycles have a frequency of about 3-4 years, and it is the amplitude (size or height) of these peaks that make up this indicator. Rodent cycles, both their regularity and their amplitude, are negatively affected by many known and unknown drivers. The most important anthropogenic driver is climate change, with warmer and wetter winters causing rodent declines due to the effect that this has on the snow cover.  
```

We get the data from the Norwegian Nature Index database [@nater_naturindeks_2025] where also the X~100~ reference values are stored.
These represent the predicted  mean number of rodents in peaks year over a 10 year period, given minimal anthropogenic influence. The reference values are also based on expert opinion, but informed by extensive empirical data. 

3. **Interpretation**:
<!-- Add a very short description of how the indicator values should be interpreted. Here's an example for a bumblebee indicator: An indicator value of 1 signifies an intact bumblebee community. Values decreasing toward zero reflect negative ecological change, specifically the decline of species expected to be present. This metric does not account for potentially positive changes, such as increases in species' abundance or the arrival of new species. Tip: Add text on the line directly following the subheader, and don't introduce blank lines. This way the whole paragraph renders as a block quote. -->

> An indicator value of 1 means that rodent numbers in peak years are as numerous, or more, than what we expect under the reference condition.

Reviewer: ```suggestion
An indicator value of 1 means that rodent numbers in peak years are as numerous, or more than what we expect under the reference condition.
```

Values decreasing toward zero indicates that the average rodent peaks are diminishing.

4. **Results and development needs**:
<!-- Briefly, what does the indicator say about the ecosystem condition. What is the current status of  the metric (can it be used or is it still in development), and what immediate tasks remain to improve the indicator or to make the indicator operational -->

> The indicator value for Norway as a whole in 2024 is 0.28 [95% CI = 0.26-0.30]. This is extremely low and reflects the deminisjong effect of rodent cycles on Norwegian alpine ecosystems. Development needs include an integration of the current monitoring time series, with the old time series based on trap monitoring.     

Reviewer: ```suggestion
The indicator value for Norway as a whole in 2024 was 0.28 [95% CI = 0.26-0.30]. This is extremely low and reflects the deminishing effect of rodent cycles on Norwegian alpine ecosystems. Development needs include an integration of the current monitoring time series, with the old time series based on trap monitoring.     
```

Dataene hentes fra databasen for Naturindeks for Norge [@nater_naturindeks_2025], hvor også referanseverdiene X~100~ er lagret.
Disse representerer det forventede gjennomsnittlige antallet smågnagere i toppår over en 10-årsperiode under minimale menneskeskapte påvirkninger. Referanseverdiene er også basert på ekspertvurderinger, men støttes av omfattende empiriske data.

3. **Tolkning**: 
En indikatorverdi på 1 betyr at smågnagerbestandene i toppår er like store, eller større, enn det vi forventer under referansetilstanden.
Verdier som nærmer seg null indikerer at toppene i smågnagersyklusene blir svakere.

4. **Resultater og utviklingsbehov**: 
Indikatorverdien for hele Norge i 2024 er 0,28 [95 % KI = 0,26–0,30]. Dette er svært lavt og gjenspeiler den reduserte effekten smågnagersyklusene har på norske alpine økosystemer. Utviklingsbehov inkluderer integrering av de nåværende overvåkingsseriene med de eldre tidsseriene basert på felleovervåking.

## 2. About the underlying data

<!--# Give a general introduction the datasets you have used, and provide more specific details in the sub-chapters.  -->

> The data comes from various trap minitoring time series, especially TOV [@Framstad2017] and selected time series from [@ehrich2020documenting]. The 2024 estimate is based on expert opinion [@nater_naturindeks_2025] . See further documentation on the [Norwegian Nature Index web site for this indicator](https://www.naturindeks.no/Indicators/smaagnagere_fjellbestander). 

Reviewer: ```suggestion
The data comes from various trap monitoring time series, especially TOV [@Framstad2017] and selected time series from [@ehrich2020documenting]. The 2024 estimate is based on expert opinion [@nater_naturindeks_2025] . See further documentation on the [Norwegian Nature Index web site for this indicator](https://www.naturindeks.no/Indicators/smaagnagere_fjellbestander). 
```

### 2.1 Spatial and temporal resolution and extent

<!--# Describe the temporal and spatial resolution and extent of the data used -->
Indicator values are available for the different assessment years in  _Naturindeksen_: 1950 (expert opinion-based estimate), 1990, 2000, 2010, 2014, 2019 and 2024 (also expert opinion-based). 

<!-- The indicator is calculated for eight carnivore regions in Norway. These may overlap with one or more of the geographic regions that we use here. Here we report the values for five geographic regions, and for Norway as a whole.  -->

### 2.2 Sampling design

<!-- How was the data sampled? Describe potential sampling biases -->

> WHen empirical data is ised, these are mostly from trap monitoring time series. Some qualitative data sources are used as well. See more information on the [Naturindeks web site](https://www.naturindeks.no/Indicators/smaagnagere_fjellbestander) and the workflow documentation found [here](https://naturindeks.nina.no/docs/ae75dc3a-2152-4365-8944-cc1d6e7a7f03.pdf).  

Reviewer: ```suggestion
When empirical data is used, these are mostly from trap monitoring time series. Some qualitative data sources are used as well. See more information on the [Naturindeks web site](https://www.naturindeks.no/Indicators/smaagnagere_fjellbestander) and the workflow documentation found [here](https://naturindeks.nina.no/docs/ae75dc3a-2152-4365-8944-cc1d6e7a7f03.pdf).  
```

### 2.3 Original units

<!--# What are the original units for the most relevant  variables in the data-->

> The original units of the raw data is *the average size of population peaks over a 10 year period, meased as captured per 100 capture-days*.

Reviewer: ```suggestion
The original units of the raw data is *the average size of population peaks over a 10 year period, measured as captured per 100 capture-days*.
```

For example, and indicator called 'Trenching in mires' could be made to represent an ecosystem characteristic 'Intact hydrology'. The term 'characteristic' is used similar to the term 'criteria' in Multiple Criteria Decision Making.  

-->

The ecosystem characteristic represented in this indicator is an ecosystem where ecosystem functions are stable (or at least predictable) and intact, and where tropic interactions occur as they would uder the reference condition. 

### 3.3 Collinearities with other indicators

<!--# Describe known collinearities with other metrics (indicators or variables) that could become problematic if they were also included in the same Ecosystem Condition Assessment as the indicator described here. -->

Since rodents are functional links between so many species and trophic levels, it is assumed to be a strongly correlated to several other indicator, such as a potential snow depth indicator, and the arctic fox indicator (NO_AFOX_001).

### 3.4 Impact factors

<!--# Describe the main natural and anthropogenic factors that affecst the metric -->

> Alpine rodents are influced by both natural (known and unknown) factors, such as predation, as well as antropogenic drivers, of which climate change is the most important. Wetter and warmer winters cause the snow cover to become thinner, less stable and less insulation, which negatively affects survaval and fitness in general.    

Reviewer: ```suggestion
Alpine rodents are influced by both natural (known and unknown) factors, such as predation, as well as antropogenic drivers, of which climate change is the most important. Wetter and warmer winters cause the snow cover to become thinner, less stable and less insulation, which negatively affects survival and fitness in general.    
```

### 3.5 Spatial aggregation

<!-- Describe in word the same aggregation pathway presented in the top yaml (the metadata). If you have scaled against the reference value, it is especially important to know if you did any avergaing or summations etc. beforehand. An example text for a rather simple case could be: "The variable is scaled at the initial polygon level and the point estimate is a area-weighted arithmetic mean of all polygins. There is no transformation, and truncation is not needed as the indicator becomes naturally bound between 0 and 1".  -->

## 4. Reference condition and levels

### 4.1 Reference condition

<!--# Define the reference condition (or refer to where it is defined). Note the distinction between reference condition and reference levels 10.3897/oneeco.5.e58216  -->

> The reference condition is a moutain ecosystem with minimal human impact and a climate similar to the period 1961-1990 [@nybo_fagsystem_2017].

Reviewer: ```suggestion
The reference condition is a mountain ecosystem with minimal human impact and a climate similar to the period 1961-1990 [@nybo_fagsystem_2017].
```

#### 4.2.1 Spatial resolution and validity

<!--# 

Describe the spatial resolution of the reference levels. E.g. is it defined as a fixed value for all areas, or does it vary. Also, at what spatial scale are the reference levels valid? For example, if the reference levels have a regional resolution (varies between regions), it might mean that it is only valid and correct to use for normalising local variable values that are first aggregated to regional scale. However, sometimes the reference levels are insensitive to this and can be used to scale variables at the local (e.g. plot) scale. 

 -->
 The reference values vary between 10 regions in Norway.

## 5. Uncertainties
### 5.1 General uncertainties
<!--# Describe the main reasons why you might not trust 100% the final value to represent the given ecosystem characteristic for the give ecosystem type. What aspects of this condition metric are most sensitive to subjective decitions? Is the spatial sampling sufficient? Does the choice of modeling framework affect the values? Is the metric relevant do describe ecosystem condition for the give ecosystem type?  -->

> One main source of uncertainty arises from the discontinuous time series, since the minitoring switched from trap minitoring to camera monitoring in 2020. There is also an issue with low spatial representation in the data pre 2020.  

Reviewer: ```suggestion
One main source of uncertainty arises from the discontinuous time series, since the monitoring switched from trap minitoring to camera monitoring in 2020. There is also an issue with low spatial representation in the data pre 2020.  
```

The reported uncertainties reflect the uncertainties around the true parameter means, i.e. they are inferrential uncertainties for the true value of the indicator for the given area and time combination.  

## 6. References

<!--# You can add references manually or use a citation manager and add intext citations as with crossreferencing and hyperlinks. See https://quarto.org/docs/authoring/footnotes-and-citations.html -->

::: {#refs}
:::

## 7. Import and prepare datasets

<!--# All the data import and preparation should be done in this chapter, before the analysis chapter. As a minimum, add code for importing data, and describe or show the data structure. You may also explain certain variables, what they represent, and their distributions -->

### 7.1 Naturindeks

<!--# Import and prepare the first dataset for analysis. Change the header from Dataset A to the name of the actual dataset. -->

We download the dataset from the Naturindeks database using the API. 

> The function giver read-only access without requiring a username and password.

Reviewer: ```suggestion
The function gives read-only access without requiring a username and password.
```

# renv::install("ChloeRN/NIcalc@include_ReadOnly_access")
# library(NIcalc)

#myYears <- as.character(c(1990, 2020, 2010, 2014, 2019, 2024))

# Import data set
roedImport <- NIcalc::importDatasetApi(
  eco = NULL,
  indic = c("Smågnagere - fjellbestander"),
  year = c("2010", "2014", "2019", "2024")
)
saveRDS(roedImport, here::here("data/roedImport.rds"))
```

```{r}
#| include: false
roedImport <- readRDS(here::here("data/roedImport.rds"))
```

## 8. Spatial units

<!--# Describe the spatial units that you rely on in your analyses. Highlight the spatial units (the resolution) that the indicator values should be interpreted at. Potential spatial delineation data (maksing layers etc.) should be introduced under chapert 7. We recommend using the SEEA EA terminology of Basic Spatial Units (BSU), Ecosystem Asses (EA) and Ecosystem Accounting Area (EAA). -->

According to the Naturindeks terminology, the 430 municipalities are called `BSunits`. In the current dataset, only 163 mountain municipalities are included. 

> The indicator data is available at the scale of 10 `ICunits`. These are unique contellation of municipalities and/or counties. 

Reviewer: ```suggestion
The indicator data is available at the scale of 10 `ICunits`. These are unique collection of municipalities and/or counties. 
```

Reviewer: Not sure what the right word is here

Author: I think constellations?, but combinations is probably better

Author: I will fix in later commit...
